tests/test_helpers.py

```diff
@@ -115,6 +115,14 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    (["helper", "finish_prum", "20sb_firstprojectum", "--end_date", "2020-07-01"],
+     "20sb_firstprojectum status has been updated to finished\n"
+     ),
+    (["helper", "finish_prum", "20sb"],
+     "Projecta not found. Projecta with similar names: \n"
```

Comment:
"Projectum not found. Projecta with similar names: \n"

Reply:
done!